src/userscores/view/recentscoresmodel.cpp

```diff
@@ -114,7 +115,8 @@ void RecentScoresModel::updateRecentScores(const MetaList& recentScoresList)
         obj[SCORE_TITLE_KEY] = !meta.title.isEmpty() ? meta.title : meta.fileName.toQString();
         obj[SCORE_PATH_KEY] = meta.filePath.toQString();
         obj[SCORE_THUMBNAIL_KEY] = meta.thumbnail;
-        obj[SCORE_TIME_SINCE_CREATION_KEY] = DataFormatter::formatTimeSinceCreation(meta.creationDate);
+        obj[SCORE_TIME_SINCE_CREATION_KEY] = DataFormatter::formatTimeSince(meta.creationDate);
```

Comment:
I left this in case it's decided it would be useful to have available (e.g. via a tooltip) in future.

Reply:
It is better to remove the unused field. We can easily add it later if we need it. But at the moment, we can slightly simplify this code

Reply:
Done